.github/ISSUE_TEMPLATE/performance_problem.yml

```diff
@@ -0,0 +1,59 @@
+name: Performance problem
+about: Report performance problems or areas of concern
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Before reporting performance problems here, please make sure you are on the latest version of Paper.
+        Forks of Paper receive no support here. If you are using a fork, please make sure that this problem also affects Paper.
+  - type: input
+    attributes:
+      label: Timings or Profile link
+      description: We ask that all timings/profiles are a link, not a screenshot. Screenshots inhibit our ability to figure out the real cause of the issue.
+      placeholder: "Example: https://timings.aikar.co/?id=6b48586fbbdd48e585ca0ebb07c59dd0"
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Description of issue
+      description: If applicable, please describe your issue.
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Plugin list
+      description: All plugins running on the server.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Please include links to your server's config files.
+      description: We need bukkit.yml, spigot.yml, paper.yml, and server.properties. Please use a paste site like https://paste.gg.
```

Comment:
Does GH support code entries? If so, we could just have them paste the files in here. Sure, it'll be a little worse to read through, but a huge boost in how easy it is to fill!

Reply:
Our choices are textarea or label, depends on whatever's easier. We _can_ set an initial value and say "put your code between the backticks", something like:

Please paste your log between the \`s. 

\`\`\`
[code here]
\`\`\`

With a placeholder of "Please don't remove the backticks :("

Reply:
Let's do both! Hopefully they either paste their paste.gg link in the code block, or better, they paste the stacktrace in it.

Reply:
I can do that for the others, but do we need it for this one if we're already asking for timings, which has the configs there?

Reply:
Didn't even think of which type it was when commenting :p But it would be preferable, as timings expire fairly quickly.

Reply:
Should be good now. Re-review if needed.